frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -190,6 +150,10 @@ const TranslationPage = () => {
           </div>
         </div>
       </div>
+      <Autosave
+        storylines={Array.from(changedStoryLines.values())}
```

Comment:
It's a shame that we have to set up an additional state `changedStoryLines` when it _feels_ like we could figure out the state changes and debounce with just `TranslatedStoryLines`. But I can't think of a different way to do this :// It's not a big deal since @jennchenn is likely going to need to do something similar like this anyways, but let me know if you have any thoughts.

Reply:
Yep, I couldn't think of a way because the UI is being updated at a different rate than the backend now so TranslatedStoryLines needs to be updated immediately. 

I was trying to imagine a world where the Autosave component could "cache" the story lines to "diff" what needs to be updated in each request, but even if that worked, it would need to iterate through the entire array to find out what changed, I think???

Reply:
"cache"ing would be interesting - I'm not sure if saving or diff-ing old state is an ugly react pattern. AutoSave's "cache" state would be reliant on the previous props it received, but accessing "previous props" certainly isn't a react-like pattern since the entire component has been rerendered